frontend/src/views/web/challenge-create.html

```diff
@@ -1,3 +1,16 @@
+<div ui-view="" class="ng-scope" style="">
+  <div class="light-dark-container ng-hide ng-scope" ng-show="isLoader" sim-loader="">
+  <div class="loader-container" id="loader" style="low-screen display: none;">
+      <div class="text-white center w-300 loader-title"></div>
+      <div class="loader">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+      </div>
+  </div>
+</div>
 <section class="ev-md-container text-center ">
```

Comment:
@HargovindArora For the faded screen you can create the structure like:
```
<section>
<div class="row">
<div class="col s12 m12">
<div class="show-loader">
</div>
</div>
</div>
</section>
```
then in the controller you can add the `low-screen` class to `show-loader`. Please let me know if you still have doubts.